profiler/options.go

```diff
@@ -158,6 +159,9 @@ func defaultConfig() (*config, error) {
 	if v := os.Getenv("DD_API_KEY"); v != "" {
 		WithAPIKey(v)(&c)
 	}
+	if v := os.Getenv("DD_PROFILING_AGENTLESS"); v == "true" {
```

Comment:
Is it worth using https://golang.org/pkg/strconv/#ParseBool ?

Reply:
It wouldn't hurt, but this option mostly exists for alignment with other the profiling libraries for other languages. For Go this option is undocumented and not supposed to be used. So I'd be okay with keeping it simple here ...

Reply:
+1 on parsebool; this option or some version of it hopefully becomes documented at some point.

Reply:
Let's use this then: https://github.com/DataDog/dd-trace-go/blob/v1/internal/env.go#L13-L15. We're using it elsewhere in the module too. People will expect a consistent experience.

Reply:
ec0b13074e5a5b19b0974c4ef93f55b229cfa6cf